prism/src/main/java/org/apache/falcon/service/EntitySLAMonitoringService.java

```diff
@@ -303,14 +303,14 @@ public void init() throws FalconException {
 
         freq = StartupProperties.get().getProperty("entity.sla.lookAheadWindow.millis", "900000");
         lookAheadWindowMillis = Integer.parseInt(freq);
-        LOG.debug("No old state exists at: {}, Initializing a clean state.", filePath.toString());
+        LOG.info("Initializing EntitySLAMonitoringService from ", filePath.toString());
```

Comment:
filePath is no more required and it doesn't initialize anything from the filePath, this line along with the filePath variable can be removed.